R/type-vctr.R

```diff
@@ -290,7 +288,11 @@ as.character.vctrs_vctr <- function(x, ...) {
 
 #' @export
 as.list.vctrs_vctr <- function(x, ...) {
-  vec_cast(x, list())
+  if (vec_is_list(x)) {
+    x
```

Comment:
So a classed list-like `vctrs_vctr` object would be returned as is, with the class still attached? I feel like I would expect a bare list to be returned, like `vec_chop()` gives?

Reply:
Definitely.